orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/ProjectController.groovy

```diff
@@ -63,9 +66,9 @@ class ProjectController {
       statuses: (statuses.split(",") as Collection)
     )
 
-    def allPipelines = rx.Observable.merge(pipelineConfigIds.collect {
+    def allPipelines = pipelineConfigIds.collect {
```

Comment:
is this a situation where we previously would have been doing multiple ops in parallel (on Schedulers.io()) but now are doing it linearly?

Reply:
Hmm, yes. I got a little carried away here. I'll revert this change.